ssl/statem/extensions.c

```diff
@@ -1242,6 +1242,13 @@ int tls_psk_do_binder(SSL *s, const EVP_MD *md, const unsigned char *msgstart,
     const char *label;
     size_t bindersize, labelsize, hashsize = EVP_MD_size(md);
     int ret = -1;
+    int usepskfored = 0;
+
+    if (external
+            && s->early_data_state == SSL_EARLY_DATA_CONNECTING
+            && s->session->ext.max_early_data == 0
```

Comment:
I guess the spec does let you send multiple PSK+binders, sure.
I a little bit wish we had a better way to be consistent with "use the first PSK for early data" than to just hardcode everywhere that the ticket-based PSK goes first, but I don't have any concrete suggestions for achieving that.